packages/mdc-elevation/_mixins.scss

```diff
@@ -46,6 +47,113 @@
   }
 }
 
+///
+/// Called once per application to set up the global default elevation styles.
+///
+@mixin mdc-elevation-overlay-common($query: mdc-feature-all()) {
+  $feat-animation: mdc-feature-create-target($query, animation);
+  $feat-structure: mdc-feature-create-target($query, structure);
+
+  @include mdc-elevation-overlay-selector_ {
```

Comment:
mdc elevation layer doesn't seem to have z-index value. how is this stacked in relation to ripple and other inner elements of a component?

Reply:
From discussion with design, the overlay appears directly above the surface but below the ripple.

Reply:
Will include this information in the docs demo

Reply:
Docs update